src/plugins/visualizations/server/migrations/visualization_saved_object_migrations.ts

```diff
@@ -945,6 +945,31 @@ const hideTSVBLastValueIndicator: SavedObjectMigrationFn<any, any> = (doc) => {
   return doc;
 };
 
+/**
+ * [TSVB] Set ignore field formatting param to true for series by default
+ */
+const addTSVBIgnoreFilterFormatting: SavedObjectMigrationFn<any, any> = (doc) => {
```

Comment:
looks like you forgot about `by value` visualizations. Please add migrations for that too